.gitignore

```diff
@@ -0,0 +1,340 @@
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+##
+## Get latest from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore
+
```

Comment:
**Added the missing .gitignore** file- to avoid pushing .dll, .config and .suo in every commit from every user. _Due to this almost 55 files have changes since they had to be deleted_

Reply:
Yeah, definitely, good idea.